app/controller/Search.js

```diff
@@ -1,48 +1,54 @@
+/*eslint angular/controller-as: 0*/
+/*eslint angular/di: [2,"array"]*/
+/*eslint max-len: [2,100]*/
 /**
  * Search Controller
  */
-angular
-    .module('SolrHeatmapApp')
-    .controller('SearchCtrl', ['Map', 'HeatMapSourceGenerator', '$scope', '$http', function(MapService, HeatMapSourceGeneratorService, $scope, $http) {
-
-        /**
-         *
-         */
-        $scope.searchInput = '';
-        /**
-         *
-         */
-        $scope.onKeyPress = function($event) {
-            // only fire the search if Enter-key (13) is pressed
-            if (getKeyboardCodeFromEvent($event) === 13) {
-                $scope.doSearch();
+angular.module('SolrHeatmapApp')
+    .controller('SearchController', ['Map', 'HeatMapSourceGenerator', '$scope', '$window',
+        function(MapService, HeatMapSourceGeneratorService, $scope, $window) {
```

Comment:
`MapService` is currently not used in this class, please remove it from requires.